Rover/Parameters.h

```diff
@@ -3,10 +3,18 @@
 #include <AP_Common/AP_Common.h>
 
 #include "RC_Channel.h"
+#include <AC_Avoidance/AC_Avoid.h>
 #include "AC_Sprayer/AC_Sprayer.h"
+#include <AP_AIS/AP_AIS.h>
+#include <AP_Beacon/AP_Beacon.h>
+#include <AP_Follow/AP_Follow.h>
 #include "AP_Gripper/AP_Gripper.h"
+#include <AP_Proximity/AP_Proximity.h>
 #include "AP_Rally.h"
+#include <AP_SmartRTL/AP_SmartRTL.h>
+#include <AP_Stats/AP_Stats.h>
 #include "AP_Torqeedo/AP_Torqeedo.h"
```

Comment:
an existing issue but it's funny that we're using quotes for the torqeedo instead of angled brackets. I'm sure it's my mistake originally though and not a blocker.

Reply:
> an existing issue but it's funny that we're using quotes for the torqeedo instead of angled brackets. I'm sure it's my mistake originally though and not a blocker.

Yes - I thought, "that's odd" and moved on....